### PR TITLE
dev-qt/qtnetwork: Fix macro redefinition of QT_LINKED_OPENSSL warning

### PR DESCRIPTION
**Pull request by a17r**
Closes: https://bugs.gentoo.org/652650

### CODE REVIEW

File: dev-qt/qtnetwork/qtnetwork-5.15.9999.ebuild [hunk 1]
```diff
@@ -66,3 +66,10 @@ src_configure() {
 	)
 	qt5-build_src_configure
 }
+
+src_install() {
+	qt5-build_src_install
+	# workaround for bug 652650
+	sed -e "/^#define QT_LINKED_OPENSSL/s/$/ $(usex ssl true false)/" \
```
Comment (Pesa) [inline dev-qt/qtnetwork/qtnetwork-5.15.9999.ebuild:73]: uhm, IIUC this issue, the macro should not be defined at all when `ssl` is disabled.

Comment (Pesa) [reply dev-qt/qtnetwork/qtnetwork-5.15.9999.ebuild:73]: what's in `/usr/include/qt5/QtNetwork/qtnetwork-config.h` if you build with `USE=-ssl`? (not on a gentoo machine right now)

Comment (Pesa) [reply dev-qt/qtnetwork/qtnetwork-5.15.9999.ebuild:73]: essentially what I'm suggesting is

```bash
if use ssl; then
    sed ... '/^#define QT_LINKED_OPENSSL/s/$/ true/' ...
fi
```

Comment (a17r) [reply dev-qt/qtnetwork/qtnetwork-5.15.9999.ebuild:73]: ...right. not much thinking went into this last night. Line is `#define QT_NO_LINKED_OPENSSL` w `USE=-ssl.`

**Review by Pesa — APPROVED** [review]
thanks

**Pull request closed without merging**